pom.xml

```diff
@@ -642,7 +642,6 @@
         <artifactId>spotless-maven-plugin</artifactId>
         <version>2.12.2</version>
         <configuration>
-          <ratchetFrom>master</ratchetFrom>
```

Comment:
Why we do not need this?

Reply:
Now that we format all packages, we do not need to find any diffs to check format

This simplifies our presubmits across branches